data_model/clusters/MeterIdentification.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0"?>
```

Comment:
How was this added? data_model files are managed by a script and should be included only as part of script execution (i.e. from parsing asciidoc files)

Reply:
In the image and likeness of other clusters